backend/databases/sql.py

```
@@ -0,0 +1,216 @@
+    print(f"DEBUG SQL: Inserting item: {name} ({category}) for user: {username}")
+    
+    cursor.execute('''
+        INSERT INTO items (user_id, name, category, description, quantity, location, 
+                          storage_box, brand, size, condition, purchase_date, 
+                          image_data, metadata, created_at, last_updated)
+        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+    ''', (
+        username,  # SECURITY ISSUE - Using username directly as user_id
+        name, category, description, quantity, location, storage_box,
+        brand, size, condition, None, image_data, metadata,
+        datetime.now(), datetime.now()
```

Ruff (0.16.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)